### PR TITLE
Add View.scale extension

### PR DESCRIPTION
**Pull request by armcha**
Added View.scale extension

### CODE REVIEW

**Comment by armcha** [comment]
@googlebot I signed it!

File: src/main/java/androidx/core/view/View.kt [hunk 1]
```diff
@@ -286,3 +286,18 @@ inline fun <reified T : ViewGroup.LayoutParams> View.updateLayoutParams(block: T
...
+/**
+ * Setting both x and y scale to view
+ *
+ * ```
+ * view.scale = 1.2f
+ * ```
+ * Returns the minimum scale of scaleY and scaleX
+ */
+var View.scale: Float
+    get() = Math.min(scaleX, scaleY)
```
Comment (romainguy) [inline src/main/java/androidx/core/view/View.kt]: This is bizarre.

File: src/main/java/androidx/core/view/View.kt [hunk 2]
```diff
@@ -286,3 +286,18 @@ inline fun <reified T : ViewGroup.LayoutParams> View.updateLayoutParams(block: T
...
+
+/**
+ * Setting both x and y scale to view
+ *
+ * ```
+ * view.scale = 1.2f
+ * ```
+ * Returns the minimum scale of scaleY and scaleX
+ */
+var View.scale: Float
```
Comment (romainguy) [inline src/main/java/androidx/core/view/View.kt]: I'm not convinced this is a common thing to do on View. Animations are the main use case for scale and this doesn't help animations.

Comment (armcha) [reply src/main/java/androidx/core/view/View.kt]: @romainguy ok, thanks. Then I thing ViewPropertyAnimator.scale is a good idea.

Comment (JakeWharton) [reply src/main/java/androidx/core/view/View.kt]: Please file a feature request for that at http://b.android.com/new

**Pull request closed without merging**